gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -160,6 +178,11 @@
 
   private static final ListFileOptions LIST_OPTIONS =
       ListFileOptions.DEFAULT.toBuilder().setFields(OBJECT_FIELDS).build();
+  /** Bhagyaa-s**/
+  private static final int DEFAULT_MAX_THREADS = 64; //16
+//  public static final String MAX_THREADS = "fs.gcs.threads.max";
+  private int executorCapacity;
```

Comment:
Do we need this var, i don't see any usages, it is just set to `16` in constructor

Reply:
We may have to change these values later, so maybe we can make a getter and setter for this that can be used by the user to modify